DuckDuckGoTests/BookmarksManagerTests.swift

```diff
@@ -1,8 +1,8 @@
 //
-//  UserText.swift
```

Comment:
Note, this file didn't get renamed. `UserText.swift` in Core got deleted and `BookmarksManagerTests` got created.